crates/testing/tests/tests_1/consensus_task.rs

```diff
@@ -498,6 +510,8 @@ async fn test_view_sync_finalize_vote() {
 /// Makes sure that, when a valid ViewSyncFinalize certificate is available, the consensus task
 /// will NOT vote when the certificate matches a different view number.
 async fn test_view_sync_finalize_vote_fail_view_number() {
+    use futures::StreamExt;
```

Comment:
Let's move this to the top of the file so we don't need to import it over and over.

Reply:
done in dec9dd6de58893f9080dde0b96285d05b2f64038